OemPkg/OverridePlatformKey/PlatformKey.h

```diff
@@ -0,0 +1,129 @@
+#ifndef PLATFORM_KEY_H_
+#define PLATFORM_KEY_H_
+
+#include <Uefi.h>

```

Comment:
The included headers in this file need to be removed.  They cause symbols to collide,